script/Deploy.s.sol

```diff
@@ -82,15 +82,33 @@ abstract contract Deploy is Common, Script {
 
     // set governor to DAO
     governor = vault721.governor();
-    delegate = address(0);
 
-    if (delegate == address(0)) {
-      _revokeAllTo(governor);
-    } else if (delegate == deployer) {
+    uint256 chainId = getChainId();
+
+    if (chainId == 42_161) {
+      // mainnet
+      if (delegate == address(0)) {
+        // if !delegate: revoke deployer && authorize governor
+        _revokeAllTo(governor);
+      } else if (delegate == deployer) {
+        // if delegate == deployer, maintain deployer auth && authorize governor
+        _delegateAllTo(governor);
+      } else {
+        // if delegate != deployer, authorize delegate && governor
+        _delegateAllTo(delegate);
+        _revokeAllTo(governor);
+      }
```

Comment:
Are you envisioning that we might have delegates initially on mainnet?

Reply:
that was logic from HAI, but i think the delegate option should actually be removed for mainnet. good point. i'll simplify that before merging this